CHANGELOG.mdx

```diff
@@ -16,6 +16,11 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.15] - 2021-11-25
+### Bugfixes
+- [#10384](https://github.com/rasahq/rasa/issues/10384): [#10384](https://github.com/RasaHQ/rasa/issues/10384): Validate regular
```

Comment:
These just link to this PR

Reply:
Yes - the issue was a rasa-x issue, which meant the link wouldn't be generated properly. In this case the instruction is to use the PR number

Reply:
Ah, right. So many PRs !